src/main/kotlin/MultiplatformHelpers.kt

```diff
@@ -25,16 +26,18 @@ private val darwinFamilies = listOf(
 )
 
 internal fun Project.applyJvmHelpers() {
-    tasks.apply {
-        task("publishForCurrentOs") {
-            dependsOn("publish")
+    tasks {
+        register("publishForCurrentOs") {
+            if (kord.commonHost.get().isCurrent()) {
+                dependsOn("publish")
+            }
         }
 
-        task("publishForCurrentOsToMavenLocal") {
+        register("publishForCurrentOsToMavenLocal") {
             dependsOn("publishToMavenLocal")
         }
 
-        task("testOnCurrentOS") {
+        register("testOnCurrentOS") {
             dependsOn("test")
```

Comment:
add that check here too

Reply:
ok, why though?

Reply:
We don't want to run those tests 3 times